### PR TITLE
masscan: add Polish translation

### PR DESCRIPTION
**Pull request by Kisielos10**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/common/masscan.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Masscan
+
+> Bardzo efektywny skaner sieci.
+> Najlepiej używać z podwyższonymi uprawnieniami. Żeby sprawdzić kompatybilność z Nmap'em, użyj komendy `masscan --nmap`.
+> More information: <https://github.com/robertdavidgraham/masscan>.
```
Comment (kbdharun) [inline pages.pl/common/masscan.md:5]:
```suggestion
> Więcej informacji: <https://github.com/robertdavidgraham/masscan>.
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. The translation LGTM, I have a suggestion for this page.

**Comment by Kisielos10** [comment]
Agree! A mistake by me. Committed the suggestion.

File: pages.pl/common/masscan.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Masscan
```
Comment (navarroaxel) [inline pages.pl/common/masscan.md:1]: This should be the name of the terminal command
```suggestion
# masscan
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**